Lib/pathlib.py

```diff
@@ -785,7 +785,7 @@ class Path(PurePath):
     """
     __slots__ = ()
 
-    def __new__(cls, *args, **kwargs):
+    def __new__(cls, *args):
```

Comment:
Can you make this raise a warning when `kwargs` is set with a plan to remove it in Python 3.14? You can use `warnings._deprecated()` for this. That way folks have time to update their code instead of breaking instantly when they upgrade to Python 3.12.

See https://peps.python.org/pep-0387/ for Python's backwards-compatibility policy and why we need to put in a warning.

Reply:
Makes sense, fixed